tests/unit/lib/telemetry/test_metric.py

```diff
@@ -227,8 +230,11 @@ def real_fn():
             "Measured duration must be in milliseconds and greater than equal to the sleep duration",
         )
 
+    @patch("samcli.lib.telemetry.metric._get_stack_trace")
     @patch("samcli.lib.telemetry.metric.Context")
-    def test_must_record_user_exception(self, ContextMock):
+    def test_must_record_user_exception(self, ContextMock, StackTraceMock):
```

Comment:
Suggest to match name casing with the method/class being patched (i.e. method -> snake_case, class -> CamelCase)

```suggestion
    def test_must_record_user_exception(self, ContextMock, get_stack_trace_mock):
```